src/frontend/screens/Library/components/InstallModal/index.tsx

```diff
@@ -198,6 +203,26 @@ export default React.memo(function InstallModal({
     />
   )
 
+  let accessCodeContent = null
+
+  if (channelRequiresAccessCode && runner === 'hyperplay') {
+    if (!isSignedIn) {
+      accessCodeContent = (
+        <AlertCard
+          showClose={false}
+          title={t('installModal.loginRequired', 'Login Required')}
+          message={t(
+            'installModal.loginRequiredMessage',
+            'You need to be logged into HyperPlay to enter your access code and install this game. '
+          )}
+          variant="warning"
+        />
```

Comment:
Could we make this a shared component to reuse in `src/frontend/components/UI/GameUpdateDialog/index.tsx`?

Reply:
yup, good call